Makefile

```diff
@@ -2,6 +2,8 @@ TARGET ?= skycastle
 CONCOURSE_URL ?= http://concourse.suse.de/
 GITHUB_ID ?= c29a4757ff0e43c25610
 GITHUB_TEAM ?= 'SUSE/Team Alfred'
+CONCOURSE_SECRETS_FILE ?= ../cloudfoundry/secure/concourse-secrets.yml.gpg
```

Comment:
Might want a `${CURDIR}` here, in case we run some of the things from other working directories.